### PR TITLE
chore(travis): remove travis in favour of Teamcity

### PR DESCRIPTION
**Pull request by kavithamadavaraj**
- removed travis.yml file
- introduced docker based builds for managing the python version and dependancies better.
- Added Make development utilities
- update Readme with TC Badge and make commands usage 

ref: TPD-22339

### CODE REVIEW

File: .travis.yml
Comment (kavithamadavaraj) [inline .travis.yml:1]: Removed this file and added these commands to ci/entrypoint.sh

File: aiohttp_jwt/__init__.py
Comment (kavithamadavaraj) [inline aiohttp_jwt/__init__.py:1]: Formatting changes across all the files in the folder `aiohttp_jwt` & `tests`

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	$(call print_h1,"COMPILING","REQUIREMENTS","USING","RSA")
+	docker run --entrypoint= --rm --tty --interactive --env SSH_AUTH_SOCK=/run/host-services/ssh-auth.sock -v /run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock  -v ~/.ssh/id_rsa:/id_rsa:ro -v ~/.ssh/id_rsa.pub:/id_rsa.pub:ro -v ${PWD}:/tenplatform/aiohttp-jwt tenproduct/aiohttp_jwt_3_6 sh -c "ssh-add /id_rsa && pip-compile --no-header --output-file=requirements.txt"
+	$(call print_h1,"REQUIREMENTS","COMPILED")
+
+#------------------------------
+# code
+#------------------------------
+
+.PHONY: lint
+lint: build-python-3.6
```
Comment (tinger2) [inline Makefile:76]: are we really supporting python 3.6?

**Review by tinger2 — APPROVED** [review]
LGTM

File: Makefile (hunk 1 above)
Comment (kavithamadavaraj) [reply Makefile:76]: This package only supports 3.6 as it is right now and almost all tests failed with other versions. (outdated dependancies may be)  🤷‍♀️

**Review by antonhrybten — APPROVED** [review]

**Pull request merged**